cdap-ui/app/features/tracker/tracker.less

```diff
@@ -545,6 +544,197 @@ body.state-tracker-help {
   }
 }
 
+body.state-tracker-integrations {
+  background-color: white;
+
+  .integration-item {
+    margin: 0;
+    padding: 10px 25px;
+    border-bottom: 1px solid @border-color;
+    height: 80px;
+    width: 100%;
+    display: table;
```

Comment:
[Comment](https://github.com/caskdata/cdap/pull/5317/files#r56508804) same here to.